lib/internal/modules/cjs/loader.js

```diff
@@ -995,10 +995,14 @@ function wrapSafe(filename, content, cjsModuleInstance) {
     });
   }
   let compiled;
+  let filenameUrl = filename;
+  try {
+    filenameUrl = normalizeReferrerURL(filename);
```

Comment:
likewise, if we can instead test for filename patterns that would make `normalizeReferrerURL` throw, it might be better than a catch all `try`/`catch`.